en/appendices/4-1-migration-guide.rst

```diff
@@ -72,6 +72,19 @@ Console
   ANSI escape codes for colours. See `no-color.org <https://no-color.org/>`__
   for more information.
 
+Error
+-----
+
+* ``debug()`` and ``Debugger::printVar()`` now emit styled HTML, or ANSI styled
```

Comment:
now emit HTML styled output in web contexts and ANSI styled output in CLI contexts.